Code/GraphMol/Fingerprints/Fingerprints.h

```diff
@@ -146,6 +146,7 @@ RDKIT_FINGERPRINTS_EXPORT ExplicitBitVect *PatternFingerprintMol(
     std::vector<unsigned int> *atomCounts = nullptr,
     ExplicitBitVect *setOnlyBits = nullptr,
     bool tautomericFingerprint = false);
+const std::string PatternFingerprintMolVersion = "1.0.0";
```

Comment:
A string containing major, minor and patch levels seems overkill for the pattern fingerprinting version.   I think an integer number would be sufficient and easier to parse.

If you want to use the current version string maybe split out the version numbers, so we can guarantee the format in the future:

```c++
const int major = 1;
const int minor = 0;
const int patch = 0
const std::string PatternFingerprintMolVersion = to_string(major)+'.'+ .... ;
```

Reply:
> A string containing major, minor and patch levels seems overkill for the pattern fingerprinting version. I think an integer number would be sufficient and easier to parse.

Agreed, but this is consistent with the other version numbers that we have for toolkits and descriptors. It's worth thinking about how to change that in order to make it easier to parse/handle, but that's a longer conversation. Maybe that kind of change would be something for the next release cycle

Reply:
Makes sense.